dockers/actions-plan-preview/main.go

```diff
@@ -109,8 +120,22 @@ func main() {
 		}
 	}
 
+	// find comment before we send
```

Comment:
nit: just to be unified, use a capital letter at the beginning of a sentence.

```suggestion
	// Find comments we sent before
```